drivers/at86rf215/at86rf215_netdev.c

```diff
@@ -176,9 +176,6 @@ static int _send(netdev_t *netdev, const iolist_t *iolist)
         at86rf215_tx_exec(dev);
     }
 
-    /* store successfully sent number of bytes */
-    dev->tx_frame_len = len;
```

Comment:
Where is dev->tx_frame_len now set? It is still used on confirm-send

Reply:
It already gets set in `at86rf215_tx_load()`